orchagent/orchdaemon.cpp

```diff
@@ -339,7 +339,11 @@ bool OrchDaemon::init()
         CFG_FLEX_COUNTER_TABLE_NAME
     };
 
-    m_orchList.push_back(new FlexCounterOrch(m_configDb, flex_counter_tables));
+    auto* flexCounterOrch = new FlexCounterOrch(m_configDb, flex_counter_tables);
+    m_orchList.push_back(flexCounterOrch);
+
+    gDirectory.set(flexCounterOrch);
+    gDirectory.set(gPortsOrch);
```

Comment:
From the change in this PR, I am not sure why adding gPortsOrch to directory is necessary. Can you elaborate?

Reply:
It is not directly related to this PR, but good to have for future use, instead of using 'extern' on each class

Reply:
@yxieca can you please review?